app/controllers/form_controller.rb

```diff
@@ -109,11 +106,19 @@ def responses_for_page(page)
   end
 
   def find_resource
-    @lettings_log = current_user.lettings_logs.find_by(id: params[:id])
+    @log = if params.key?("sales_log")
```

Comment:
Would prefer to be more explicit about the two expected cases and error handling here but just a nit